wasm/src/main/scala/wasm4s/WasmContext.scala

```diff
@@ -55,26 +55,27 @@ trait ReadOnlyWasmContext {
       IRTypes.ArrayType(typeRef)
   }
 
-  /** Collects all methods declared, inherited, and mixed-in by the given class, super-class, and
-    * interfaces.
+  /** Collects all methods declared and inherited by the given class, super-class.
     *
     * @param className
     *   class to collect methods from
     * @param includeAbstractMethods
     *   whether to include abstract methods
     * @return
-    *   list of methods in order that "collectMethods(superClass) ++ methods from interfaces ++
-    *   methods from the class"
+    *   list of methods in order that "collectVTableMethods(superClass) ++ methods from the class"
```

Comment:
I thought we need to collect methods from interfaces too, so we can collect the method definitions in `trait`s such as ↓,

```scala
trait Foo {
  def foo: Int = 1
}
```

but this isn't needed?
Is this because those method definitions in traits will be somehow merged into the class that implements that trait?